sqlmesh/core/config/connection.py

```diff
@@ -294,7 +294,7 @@ def create_engine_adapter(self, register_comments_override: bool = False) -> Eng
         if self.database:
             if isinstance(self, MotherDuckConnectionConfig):
                 data_files.add(
-                    f"md:{self.database}"
+                    f"md:{self.database.replace('md:', '')}"
```

Comment:
What case do we account for here with this replacement?

Reply:
This would be a friendly enhancement to allow the user to specify `database` with the `md:` prefix as well as without

Reply:
IMHO, i'd prefer to have a consistent format what database is. `md:` is a URI and not a database name